source/contracts/reporting/Market.sol

```diff
@@ -159,6 +160,7 @@ contract Market is DelegationTarget, ITyped, Initializable, Ownable, IMarket {
             updateTentativeWinningPayoutDistributionHash(tentativeWinningPayoutDistributionHash);
         }
         reportingWindow.updateMarketPhase();
+        Augur(controller.lookup("Augur")).logReportsDisputed(getUniverse(), msg.sender, this, uint8(ReportingState.DESIGNATED_DISPUTE), _bondAmount);
```

Comment:
I wonder if we should add a method on `Controller` named `getAugur` that just does:
```
function getAugur() external view returns (Augur) {
    return lookup("Augur");
}
```
Then we wouldn't have to have this ugly thing everywhere.  Only reason I think special casing is valuable in this case is because looking up this particular contract is _soo_ common.

Reply:
That sounds liek a good idea to me. I was bothered by this too.